dvc/utils/table.py

```diff
@@ -21,7 +21,7 @@ class Column(RichColumn):
 
 
 class Table(RichTable):
-    def add_column(  # pylint: disable=arguments-differ
+    def add_column(  # type: ignore[override] # pylint: disable=arguments-differ,arguments-renamed # noqa: E501
```

Comment:
The linter is raising an appropriate error message. Also this should be upstreamed, see: https://github.com/iterative/dvc/pull/5753#issuecomment-811822900.

Reply:
@skshetry Indeed, should've looked deeper into that. @pmrowla any plans to upstream it?